calculate_average.js

```diff
@@ -0,0 +1,14 @@
+
+const calculateAverage = function calculateAverage(grades) {
+
+  let total = 0;
+  for (let i = 0; i < grades.length; i++) {
```

Comment:
We have a variable, total (on line 4) that has the same value as the variable i in this line. Can we consider updating the variables so they are consistent, as it appears like total and i are referring to the same value (the sum) in calculating the average